### DISCUSSION TITLE
Task 1 - Part 4: .h5 files not found

### DISCUSSION THREAD

**Discussion by MGriffiths96** [fusion-energy/neutronics-workshop#223] [Q&A] [answered]
Hello, 

First of all I'd just like to say thank you for producing this workshop and making it so easy to access, it's been great to learn some of the basics of OpenMC in my own time.

I am currently going through task 1, part 4 and have come across an issue. For both the W-186 and Fe-56 sections I am getting a FileNotFoundError. I have put the screenshot of the error for W-186 below (the error for Fe-56 is basically the same but with the corresponding .h5 file). 

![image](https://github.com/fusion-energy/neutronics-workshop/assets/56959369/e7be617a-dd04-4a60-92a8-667421911ca8)

I didn't want to flag this as an issue as I realise this may be an oversight on my end. I am using Docker 24.0.4 on Ubuntu and pulled the workshop image this morning so I am fairly certain everything is up to date.

Do you have any suggestions on where this issue is coming from and how I might resolve it? If there is anything else you need from me relating to this, please do let me know. 

Thank you in advance.

**Comment by shimwell** [comment]
thanks for reporting , I shall look into this asap. I think this is a bug with the location of these wmp cross sections but I shall confirm

**Reply by MGriffiths96** [reply]
Thank you!

**Comment by shimwell** [comment, marked as answer]
It looks like I am missing a ```/``` from the path

Try replacing this line
```h5_file = f"WMP_Library/074186.h5"```

with this line
```h5_file = f"/WMP_Library/074186.h5"```

**Reply by MGriffiths96** [reply]
That's working for me. Thank you!

**Reply by shimwell** [reply]
I shall fix this for all in the next release of the docker image. Thanks for reaching out

**Reply by shimwell** [reply]
fixed in develop branch already https://github.com/fusion-energy/neutronics-workshop/pull/224
